radix-engine/src/transaction/state_update_summary.rs

```diff
@@ -22,8 +22,10 @@ pub struct StateUpdateSummary {
     pub new_components: IndexSet<ComponentAddress>,
     pub new_resources: IndexSet<ResourceAddress>,
     pub new_vaults: IndexSet<InternalAddress>,
+    /// TODO: remove
```

Comment:
Is there currently anything missing to remove this?

Reply:
Core API still depends on it. We will be able to remove them once Node-side balance change accounting is implemented.